index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://code.jquery.com/jquery-1.10.2.js"></script>
+  <link rel="stylesheet" href="style.css">
+  <title>Index</title>
+</head>
+<body class='grid-container'>
+  <div id="header-placeholder"></div>
+  <script>
+  $(function(){
+    $("#header-placeholder").load("header.html");
+  });
```

Comment:
Very cool! We'll learn other ways to re-use mark-up when we discuss React components.